src-tauri/src/northstar/install.rs

```diff
@@ -36,13 +36,13 @@ async fn do_install(
     game_path: &std::path::Path,
 ) -> Result<()> {
     let filename = format!("northstar-{}.zip", nmod.version);
-    let download_directory = format!("{}/___flightcore-temp-download-dir/", game_path.display());
+    let temp_dir = format!("{}/___flightcore-temp-dir", game_path.display());
```

Comment:
since the purpose of the directory is now to do multiple things, the name was changed.
There are many other references to `___flightcore-temp-download-dir` that may need to be adapted, but this could leave leftover files that we can no longer handle when a user did something unexpected in the past.

Should we change the name everywhere, or keep the old name and simply update the structure?

Reply:
If you update the directory name, then make sure to also update the logic called from RepairView that deletes the temp folder to delete both the one with the old name and the new name.

IMO, it would be easiest to just keep the name and have a separate PR for changing it.